Assets/Scripts/DataTypes/Edges/Anchor.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using UnityEngine;
+
+namespace DataTypes
+{
+    public struct Anchor
+    {
+        public Road parentRoad;
+        public AnchorNumber number;
+        public Vector2 position;
+        public ImmutableArray<Lane> endingLanes;
```

Comment:
Merges und Splits müssen durch Sections symbolisiert werden.

Reply:
In #14 festgehalten: https://github.com/rrickfox/traffic-sim/issues/14#issuecomment-576052655